thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js

```diff
@@ -31,14 +31,14 @@ export default Component.extend({
   //
   // external (optional)
   //
-  analysisRangeMax: moment().endOf('day'),
+  analysisRangeMax: moment().endOf('day').valueOf() + 1,
```

Comment:
what does the +1 do here? Seems very arbitrary

Reply:
yeah I could also use startOf('day').plus(1, 'day')

Reply:
@apucher .add(1, 'day') is the correct method I believe 
https://momentjs.com/docs/#/manipulating/add/